src/igvfd/audit/tissue.py

```diff
@@ -22,7 +26,11 @@ def audit_tissue_ccf_id(value, system):
 
 @audit_checker('Tissue', frame='object')
 def audit_tissue_ccf_id_nonhuman_sample(value, system):
-    '''Tissue objects must not specify a common coordinate framework identifier (CCF ID) unless the sample is from a human donor.'''
+    '''
+        audit_detail: Tissues must not specify a common coordinate framework identifier (CCF ID) unless the sample is from a human donor.
```

Comment:
Non human tissue samples are not expected to specify a common coordinate framework identifier (CCF ID).